packages/dds/merge-tree/src/perspective.ts

```diff
@@ -0,0 +1,239 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { UnassignedSequenceNumber } from "./constants.js";
+import { type MergeTree } from "./mergeTree.js";
+import { LeafAction, backwardExcursion, forwardExcursion } from "./mergeTreeNodeWalk.js";
+import { type ISegment } from "./mergeTreeNodes.js";
+
+/**
+ * Defines a side relative to an element in a sequence.
+ *
+ * @public
```

Comment:
these are all un-exported so should not have any tagging. Because they are unexported the tags do nothing, but better to not have it, as if they ever get exported the actual tagging need to be consistent